.github/workflows/bazel_worker.yaml

```diff
@@ -15,10 +16,6 @@ on:
   schedule:
     - cron: "0 0 * * 0"
 
-env:
-  PUB_ENVIRONMENT: bot.github
```

Comment:
Did you double check w/ the pub folks on this?

It was meant to avoid counting "bots" for package downloads. Not sure if it matters any more, but might be good to check.

Reply:
cc @dart-lang/dart-pub-team; I assume we're already handling pings from github action bots? I'd prefer not including this if possible - it's cruft in the file, and I assume duplicative.

Reply:
If 'CI' is on in the environment we don't send the pub metadata headers, so this should indeed be redundant.

See: https://github.com/dart-lang/pub/blob/a652ee4881d70ae18dc8ae6136166e9d1015d553/lib/src/source/hosted.dart#L194

Reply:
And for the version counts we indeed ignore all the metadata headers.

Reply:
Thanks!